### DISCUSSION TITLE
What happened to splitViewController?

### DISCUSSION THREAD

**Discussion by ohshitnotgood** [siteline/SwiftUI-Introspect#370] [Q&A] [answered]
How do we access `.introspectSplitViewController` now?

**Comment by davdroman** [comment, marked as answer]
- [NavigationSplitView](https://swiftpackageindex.com/siteline/swiftui-introspect/main/documentation/swiftuiintrospect/navigationsplitviewtype)
- [NavigationView with .columns style](https://swiftpackageindex.com/siteline/swiftui-introspect/main/documentation/swiftuiintrospect/NavigationViewWithColumnsStyleType)
